src/mpid/ch4/src/ch4r_win.h

```diff
@@ -242,6 +366,13 @@ static inline int MPIDI_CH4R_win_init(MPI_Aint length,
         MPIDI_CH4U_WIN(win, info_args).alloc_shm = 0;
     }
 
+    /* default any op */
+    MPIDI_CH4U_WIN(win, info_args).which_accumulate_ops = 0;
+    for (op_shift = MPIDI_CH4I_ACCU_MAX_SHIFT; op_shift < MPIDI_CH4I_ACCU_OP_SHIFT_LAST; op_shift++)
```

Comment:
Similar comment here. Can't we just start from zero?

Reply:
It has been changed to start from 0 (MPIDI_CH4I_ACCU_MAX_SHIFT == 0).